### PR TITLE
Setup scss!

### PR DESCRIPTION
**Pull request by nbuonin**
Here's a first pass at using node-scss. This commit edits package.json
to include the needed npm packages as well as adds script declarations
to do:
- watch the scss folder
- when a file changes, compile the scss to css, then run that css
through autoprefixer.

This commit also adds a Make target to run the above NPM script.

### CODE REVIEW

**Comment by nbuonin** [comment]
Hi @nikolas, because you have more experience with Node than I do, I thought you would be best to review this.

File: package.json [hunk 1]
```diff
@@ -4,21 +4,29 @@
...
+    "watch-scss": "node-sass --output static/css static/scss && postcss -u autoprefixer -r static/css/main.css",
+    "build-scss": "nodemon -e scss -x 'npm run watch-scss'"
   },
   "repository": {
     "type": "git",
     "url": "git://github.com/ccnmtl/histologylab-static.git"
   },
-  "author": "",
+  "author": "Center for Teaching and Learning, Columbia University",
   "license": "BSD-2-Clause",
```
Comment (zmustapha) [inline package.json]: I thought we use GPL3?

Comment (nikolas) [reply package.json]: It's up to whoever sets things up. I always use GPL3 - i think we have some libraries that are BSD or others.

**Comment by nikolas** [comment]
Cool - i made some changes and i'll merge.

**Pull request merged**